### PR TITLE
wazo-tox: add the ability to the the integration_test_timeout

### PR DESCRIPTION
**Pull request by pc-m**
setting this variable should add the INTEGRATION_TEST_TIME environment variable

### CODE REVIEW

File: roles/wazo-tox/tasks/main.yaml [hunk 1]
```diff
@@ -35,6 +35,11 @@
       WAZO_TEST_DOCKER_OVERRIDE_EXTRA: "{{ docker_sibling_results.file }}"
   when: docker_install_siblings
 
+- name: Set the INTEGRATION_TEST_TIMEOUT environment variable
+  set_fact:
+    integration_test_timeout:
+      INTEGRATION_TEST_TIMEOUT: '{{ integration_test_timeout | default(omit) }}'
+
```
Comment (pc-m) [inline roles/wazo-tox/tasks/main.yaml]: I'm not sure if I need a `when` here. If I do, I'm not sure what it should be.

**Comment by pc-m** [comment]
Note to the reviewer. What I'm trying to accomplish is to add a variable in my `zuul.yaml` that will result in the `INTEGRATION_TEST_TIMEOUT` being set to that value. The accompanying PR is here https://github.com/wazo-pbx/wazo-dird/pull/8

File: roles/wazo-tox/tasks/main.yaml (hunk 1 above)
Comment (sileht) [reply roles/wazo-tox/tasks/main.yaml]: no you don't need it

**Comment by pc-m** [comment]
recheck

**Pull request merged**